run.sh

```diff
@@ -233,6 +240,14 @@ if [[ "$EXIT_CODE" == "0" ]]; then
     }
 fi
 
+echo "Sending exit code ${EXIT_CODE} as metric."
+echo -n "transport_test.sample.exit_code:${EXIT_CODE}|g|1|#${METRIC_TAGS}"|nc -4u -w1 localhost 8125
```

Comment:
how do you see yourself using this metric?

alert if it is `> 0`?

I wonder if doing a count for "exit"/"finished" with a tag for error code would make more sense, given that it is really more of a static status code rather than a numeric

Reply:
Yeah the idea was to alert in non-zero scenarios, though I may need to adjust how this is sent for Java... it exits as 130 with SIGINT. 
We send the run/finished metrics in the samples already, but this could be a separate set of script run/end versus sample run/end.

The script fails sometimes because of docker issues, even if the sample doesn't, so that's valuable information.